wkcuber/wkcuber/__init__.py

```diff
@@ -5,9 +5,14 @@
 For information regarding the other functionalities, refer to the `README.md`.
 """
 
+import warnings
+
+from webknossos import Mag
+
 from .api.dataset import Dataset
+from .compress import compress_mag
 from .cubing import cubing
 from .downsampling import downsample_mags
-from .compress import compress_mag
-from .mag import Mag
 from .metadata import write_webknossos_metadata
+
+warnings.filterwarnings("once", category=DeprecationWarning)
```

Comment:
I guess that's the default anyways, or is it configured differently somewhere else?

Reply:
Yes, probably.